src/comments.js

```diff
@@ -1,17 +1,69 @@
 import dialogPolyfill from 'dialog-polyfill';
+import {
+  getComments, postComments,
+} from './api';
 
 const dialog = document.querySelector('#dialog');
 dialogPolyfill.registerDialog(dialog);
 
-const displayComments = () => {
+const commentsList = document.querySelector('#comments-list');
+
+const displayError = () => {
+  const commentMessage = document.createElement('li');
+  commentMessage.className = 'no-comments';
+  commentMessage.textContent = 'No Comments';
+  commentsList.append(commentMessage);
+};
+
+const displayComment = (comment) => {
+  const commentElement = document.createElement('li');
+  commentElement.className = 'comment';
+  commentElement.textContent = `${comment.creation_date} ${comment.username} : ${comment.comment}`;
+  commentsList.append(commentElement);
+};
+
+const displayComments = async (showId) => {
+  commentsList.innerHTML = '';
+
+  try {
+    const comments = await getComments(showId);
+    comments.forEach((comment) => displayComment(comment));
+  } catch (error) {
+    displayError();
+  }
+};
```

Comment:
On line 30, where you run your displayComment function. I would suggest that you create some validation to check if "comment.item_id" corresponds with the "showId" before display to make sure you are using the right information for that particular show.